develop/migration_rt.rst

```diff
@@ -1,6 +0,0 @@
-Real-Time Analytics Apps
```

Comment:
Why is this deleted?

Reply:
Part of the release checklist is to fix any sphinx build warnings. This file wasn't referenced in any menu or document, so sphinx was complaining and I removed the file.

If you prefer we can restore it and add a menu entry.

Reply:
I didn't realize this was not being displayed anywhere. Did this disappear as part of our migration section revamp? I thought it was a small blurb in that section initially. See: http://docs.citusdata.com/en/v7.3/develop/migration.html.

Was this consciously removed, or just missed? The content is pretty content-free, so I'm OK removing, but have some note to contact-us (or maybe links to our use-case guides / customer case-studies) might be helpful.

Reply:
> Did this disappear as part of our migration section revamp?

Yeah, it didn't fit in with the nice progression of MT steps, and it didn't have much information. The only reason to keep it in is to clarify that the rest of the migration content is skewed towards the MT case (the section is labeled as just "migrating and existing app" which appears more general than it is)

Reply:
Ok, then could you open up a new issue to upgrade our content? Could you also capture this deleted information in that issue (Just as context for what we used to have)?

We can discuss it outside of the 8.0 release. Thanks